### ISSUE TITLE
metadata injector will not open, keeps crashing (mac)

### ISSUE THREAD

**Issue by recreationman** [google/spatial-media#292]
even though i have chosen to allow opening, it will not open. frustrating....

**Comment by recreationman** [comment]
OS Monterey 12.7.4

**Comment by jamiesonpepper** [comment]
Are you able to use the CLI commands instead via a terminal on Mac? I'd be curious about the error if it happens there as well.

**Comment by nasklik** [comment]
Same here, on MBP M1 chip, crashes. Is there a way to give you logs or something?

**Comment by rderimay** [comment]
To get it to work on a Mac Mx:

1. Install Python 2 using pyenv
2. run gui.py from the source code

`brew install pyenv`
`pyenv install 2`
`pyenv global 2`
run this: `echo -e 'if command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init -)"\nfi' >> ~/.zshrc`
reload your shell
`cd spatialmedia`
`python gui.py`

**Comment by Teufeuleu** [comment]
> did someone get this working?

Yes. If you don't want to bother installing Docker to run the webapp, you can use python.
- clone the repo
- `cd` into it
- run `python spatialmedia -i [--stereo=(none|top-bottom|left-right)] [--spatial-audio] <input> <output>`
